src/libman/GlobalSuppressions.cs

```diff
@@ -4,10 +4,16 @@
 // Project-level suppressions either have no target or are given 
 // a specific target and scoped to a namespace, type, member, etc.
 
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1032:Implement standard exception constructors", Justification = "Custom exception types require multiple parameters to be useful")]
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Design", "CA1032:Implement standard exception constructors", Justification = "Custom exception types require multiple parameters to be useful")]
 
 // Suppress issues in code from external sources
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1064:Exceptions should be public", Justification = "<Pending>", Scope = "type", Target = "~T:Microsoft.Extensions.CommandLineUtils.CommandParsingException")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1307:Specify StringComparison", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Extensions.CommandLineUtils.CommandOption.#ctor(System.String,Microsoft.Extensions.CommandLineUtils.CommandOptionType)")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1307:Specify StringComparison", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Extensions.CommandLineUtils.AnsiConsole.WriteLine(System.String)")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1307:Specify StringComparison", Justification = "<Pending>", Scope = "member", Target = "~M:Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(System.String[])~System.Int32")]
+[assembly: SuppressMessage("Design", "CA1064:Exceptions should be public", Justification = "<Pending>", Scope = "type", Target = "~T:Microsoft.Extensions.CommandLineUtils.CommandParsingException")]
```

Comment:
The `<Pending>` was in the entries before this change, so just I'm assuming you're fine with it, just making sure since it seems like the type of thing that would nag at you :)

Reply:
Yeah, all of these are under the comment for external sources (from the Microsoft.Extensions.CommandLineUtils.Sources package)